x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/assistant_overlay/index.tsx

```diff
@@ -137,7 +139,6 @@ export const AssistantOverlay = React.memo(() => {
         hideCloseButton
       >
         <Assistant
-          lastConversation={lastConversation}
```

Comment:
`lastConversation` should not be passed as props, as this value should be retrieved from local storage to get the latest value.

Reply:
I have reverted this change as it's still needed when open chat from the rules page.